src/main/java/org/motechproject/prevac/service/impl/PrimeVaccinationScheduleServiceImpl.java

```diff
@@ -187,38 +218,6 @@ private Visit getScreeningDetails(Visit visit) {
         return null;
     }
 
```

Comment:
In the checkNumberOfPatients method is used lookup findByBookingPlannedDateClinicIdAndVisitType which is using bookingPlannedDate instead of dateProjected